### DISCUSSION TITLE
webpack5 always resolve "@" to "../../"

### DISCUSSION THREAD

**Discussion by Tasarinan** [webpack/webpack#12756] [General]
recently I added a new package like the following:

import Editor, { createEditorStateWithText, composeDecorators } from '@draft-js-plugins/editor';

but seems  after webpack parse, it shows the following error. seem it always parse @ to ../../
how to solve this issue?
Module not found: Error: Can't resolve '../../draft-js-plugins/editor'

**Comment by alexander-akait** [comment]
Please create reproducible test repo

**Reply by Tasarinan** [reply]
I have solved this issue. this issue is my mistake for one plugin of babel
